nservicebus/upgrades/5to6.md

```diff
@@ -231,12 +231,13 @@ The configuration option `DoNotRequireExplicitRouting()` has been obsoleted sinc
 
 AutoSubscription happens during the startup phase of the bus. Previous versions of NServiceBus did try to subscribe multiple times on a background thread until the subscription either succeeded or failed. When the subscription failed, an error entry was written to the log file. This version of NServiceBus changes that behavior for transports with message driven pub-sub. The subscription is tried asynchronously on the startup thread. In the case when a subscriber starts and the publisher has never created its queues, the subscriber endpoint will not start and the caller will receive a `QueueNotFoundException` indicating what went wrong.
 
+## Queue creation
 
-## Assembly scanning
+The interface `IWantQueueCreated` has been replaced by the `QueueBindings` class. There is a singleton `QueueBindings` object available via the settings API (`ReadOnlySettings` interface). For more information how to declare and create queues see [Declare and create queues](/nservicebus/extending/queuecreation.md).
```

Comment:
drop the word "singleton" it has too much bagade. just say "the instance of QueueBindings can be retrieve/accessed via"